ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template1.kt

```diff
@@ -127,13 +138,20 @@ private fun ColumnScope.Template1MainContent(state: PaywallState.Loaded) {
 }
 
 @Composable
-private fun HeaderImage(uri: Uri?) {
+private fun HeaderImage(uri: Uri?, template1Size: IntSize) {
     uri?.let {
         CircleMask {
+            val aspectRatio = template1Size.height.toFloat() / template1Size.width
+            val screenHeight = LocalConfiguration.current.screenHeightDp
             RemoteImage(
                 urlString = uri.toString(),
                 modifier = Modifier
-                    .aspectRatio(ratio = 1.2f),
+                    .conditional(aspectRatio > 1f) {
+                        aspectRatio(ratio = 1.2f)
+                    }
+                    .conditional(aspectRatio <= 1f) {
+                        fillMaxWidth().height(screenHeight.dp / 2f)
```

Comment:
Does this keep the aspect ratio? The `fillMaxWidth()` will fill the parent and the height will be reduced? So a portrait image will look squashed right?

Reply:
No it won't. However, we are using `ContentScale.Crop` here, so the image won't look squashed but it will get cut, so depending on the image it might not look great.

I agree, this might not look great in some scenarios, but usually, this shouldn't be a problem as long as the width is not much larger than the height. So, for developers presenting as a dialog, this shouldn't be an issue I think (the dialog will not use the full available width in these scenarios.

Do you have any suggestions to handle this instead? This is the screenshots change after this: https://github.com/RevenueCat/purchases-android-snapshots/pull/3

Reply:
I was missing the `ContentScale.Crop` 😄 

I was worried the image would look squashed, but it won't, so this should be fine